app/views/fileChecksProgress.scala.html

```diff
@@ -4,26 +4,35 @@
 
 @main("Checking records") {
     @defining(play.core.PlayVersion.current) { version =>
-    <div class="govuk-grid-row">
+    <div class="govuk-grid-row file-check-progress">
         <div class="govuk-grid-column-two-thirds">
 
             @progressIndicator(Messages("checkingRecords.progress"))
             @frontEndInputs(frontEndInfo)
-            <h1 class="govuk-heading-l">Checking records</h1>
-            <p class="govuk-body">We are now checking your records. It is safe to close the browser if you wish, otherwise please wait and you will be redirected automatically once complete.</p>
+            <h1 class="govuk-heading-l">Checking your records</h1>
             <input id="consignmentId" type="hidden" value="@consignmentId">
-            <div class="file-check-progress">
-                <label for="av-metadata-progress-bar" class="govuk-label">Anti-virus scanning</label>
-                <progress id="av-metadata-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.avMetadataProgressPercentage" max="100"></progress>
-            </div>
-            <div class="file-check-progress">
-                <label for="checksum-progress-bar" class="govuk-label">Validating data integrity</label>
-                <progress id="checksum-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.checksumProgressPercentage" max="100"></progress>
-            </div>
-            <div class="file-check-progress">
-                <label for="ffid-progress-bar" class="govuk-label">Identifying file formats</label>
-                <progress id="ffid-progress-bar" class="file-check-progress__progress-bar" value="@fileChecks.ffidMetadataProgressPercentage" max="100"></progress>
+            <p class="govuk-body">Please wait while your records are being checked. This may take a few minutes.</p>
+            <p class="govuk-body">The following checks are now being performed:</p>
+            <ul class="govuk-list govuk-list--bullet">
+                <li>Anti-virus scanning</li>
+                <li>Identifying file formats</li>
+                <li>Validating data integrity</li>
+            </ul>
+            <div class="govuk-notification-banner" id="file-checks-completed-banner" tabindex="-1" role="region" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" @if(!fileChecks.isComplete) {hidden}>
+                <div class="govuk-notification-banner__header">
+                    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+                        Important
+                    </h2>
+                </div>
+                <div class="govuk-notification-banner__content">
+                    <p class="govuk-notification-banner__heading">Your records have been checked</p>
+                    <p>Please click continue to see your results.</p>
```

Comment:
Could you change it to "Please click 'Continue' to see your results." as we have referred to button labels like that in other places I think. :) 
And just to stay consistent, could you add the class="govuk-body" to the p tag?

Reply:
I've changed the text and added the class to the p tag.